robot/basestation/static/js/elements/navigation/localization.js

```diff
@@ -0,0 +1,129 @@
+let rover_heading = 0
```

Comment:
Just a FYI. I recommend wrapping all JS files in `$(document).ready(()=> {code});`

Look at other JS files. Basically this only loads the JS once the whole thing is loaded. If you don't do this and you attach a jquery listener before the page is loaded, youll have a huge headache

Reply:
I tried this and the canvas simply didn't load.